test/SlowTests/Server/Documents/ETL/EtlTimeSeriesTests.cs

```diff
@@ -0,0 +1,2480 @@
+                    var shouldNotLoaded = await session.Query<Person>().ToArrayAsync();
+                    shouldNotLoaded = shouldNotLoaded
+                        .Where(u => expectedNotLoaded.Contains(u.Name))
+                        .ToArray();
+                    Assert.Empty(shouldNotLoaded);
+                }
+            }
+            catch (Exception e)
+            {
+                ThrowWithEtlErrors(src, e);
+            }
+        }
+
+        [Fact]
+        public async Task RavenEtlWithTimeSeries_WhenLoadTwoRangeAndAdd()
+        {
+            string[] collections = {"Users"};
+            const string script = @"
+var user = loadToUsers(this);
+
+user.addTimeSeries(loadTimeSeries('Heartrate', new Date(2020, 3, 10), new Date(2020, 3, 14)));
+user.addTimeSeries(loadTimeSeries('Heartrate', new Date(2020, 3, 20), new Date(2020, 3, 24)));
+"; // the month is 0-indexed
+            var ranges = new (DateTime from, DateTime to)[]
+            {
+                (new DateTime(2020, 4, 10), new DateTime(2020, 4, 14)),
+                (new DateTime(2020, 4, 14), new DateTime(2020, 4, 24)),
+            };
+            
+            var src = GetDocumentStore(_options);
+
+            try
+            {
+                var dest = GetDocumentStore();
+                
+                AddEtl(src, dest, collections, script, collections.Length == 0);
+
+                var times = new List<DateTime>();
+                for (var i = new DateTime(2020, 4, 1); i < new DateTime(2020, 4, 30); i += TimeSpan.FromDays(1))
+                {
+                    times.Add(i);
+                }
+                const string timeSeriesName = "Heartrate";
+                const string tag = "fitbit";
+                const double value = 58d;
+                var user = new User();
+
+                using (var session = src.OpenAsyncSession())
+                {
+                    await session.StoreAsync(user);
+                    foreach (var dateTime in times)
+                    {
+                        session.TimeSeriesFor(user, timeSeriesName).Append(dateTime, new[] {value}, tag);    
+                    }
+                    await session.SaveChangesAsync();
+                }
+
+                await AssertWaitForTrueAsync(async () =>
+                {
+                    using var session = dest.OpenAsyncSession();
+                    var ts = await session.TimeSeriesFor(user.Id, timeSeriesName).GetAsync();
+                    if (ts == null)
+                        return false;
+                    return ts.All(t => 
+                        (t.Timestamp >= ranges[0].from && t.Timestamp <= ranges[0].to)
+                        || (t.Timestamp >= ranges[1].from && t.Timestamp <= ranges[1].to));
+                }, interval: _waitInterval);
+            }
+            catch (Exception e)
+            {
+                ThrowWithEtlErrors(src, e);
```

Comment:
We need to handle it better - it's not a good practice to `try / catch` entire test